Minor formating changes in docToolchainConfig.groovy

MichaelRossner opened the pull request:
- replace <TAB> with <SPACE>
- add missing spaces for lists

### All Submissions:

* [n] Did you update the `changelog.adoc`?
* [n] Does your PR affect the documentation?
* [ ] If yes, did you update the documentation or create an issue for updating it?

MichaelRossner committed:
Minor formating changes in docToolchainConfig.groovy

- replace <TAB> with <SPACE>
- add missing spaces for lists

Signed-off-by: rmi2hi <michael.rossner@de.bosch.com>

PacoVK commented on docToolchainConfig.groovy:
```diff
@@ -220,10 +220,10 @@ confluence.with {
+    - gradle variables set through gradle properties (uses the 'confluenceUser' and 'confluencePass' keys)
     Often, same credentials are used for Jira & Confluence, in which case it is recommended to pass CLI parameters for both entities as
-    -Pusername=myUser -Ppassword=myPassword
+    - Pusername=myUser -Ppassword=myPassword
```
i am not sure about this change, there should not be a whitespace ;)

MichaelRossner replied:
Oops. Of course - this is not an unsorted list.

Thanks.

MichaelRossner committed:
Update docToolchainConfig.groovy

removed extra space as this is not a unsorted list but a command line parameter option

MichaelRossner committed:
Merge branch 'docToolchain:ng' into ng

PacoVK reviewed (approved):
LGMT, thanks :)
